doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/indoor-living-wall.tex

```diff
@@ -0,0 +1,162 @@
+\section{Indoor Living Wall }\label{indoor-living-wall}
+
+Indoor living walls refer to vertically built structures where plants grow soil-based or hydroponically, provide natural cooling effects through plant evapotranspiration, and enhance overall indoor environmental quality in the built environment. Pilot studies shows measurable benefits of hydroponic indoor greenery systems on reducing building cooling rates. This object mathematically describes the thermal performance of indoor living wall systems through surface heat balance as well as heat and mass balance of thermal zones where the indoor living walls are located. 
+
+\subsection{Energy Balance of Indoor Living Wall}\label{energy-balance-of-indoor-living-wall}
+
+The IndoorLivingWall object directly connects with the inside surface heat balance, zone air heat balance, and zone air moisture balance in EnergyPlus. Indoor living wall surface heat balance, which determines leaf surface temperature, takes into account convective heat transfer between indoor living walls and zone air, incident shortwave solar radiation, longwave radiation with surrounding surfaces, heat required for vaporization from ET, and heat conduction. Latent load from ET of indoor living walls contributes to indoor air moisture balance. 
+
+Plant energy balance equation:
+
+\begin{equation}
+Q_{lw-net}+Q_{sw}+h_{ip} \cdot A_ip \cdot (T_z - T_p )-\lambda \cdot A_ip \cdot ET+Q_{cond}=0          
+\end{equation}
+
+where:
+
+\(Q_{lw-net}\) is the net longwave radiation from surrounding surfaces to indoor living walls(W);
+
+\(Q_{sw}\) is the shortwave radiation on indoor living wall surface(W);
+
+\(h_{ip}\) is the convective heat transfer coefficient(W/(m\(^2\)\(^{\circ}\)C));
+
+\(T_z\) is the zone air temperature (\(^{\circ}\)C);
+
+\(T_p\) is the plant surface temperature (\(^{\circ}\)C);
```

Comment:
Throughout the document, I question the use of Celsius versus Kelvin? I find reading rates in particular in `x/C` (as opposed to `x/K`) a bit weird. Is it a conscious choice?

Reply:
Both zone air and plant surface temperatures are in Celsius (C)for the convective heat transfer. People may have different preferences although W/(m2C) and W/(m2K) are both correct. To be consistent, C is used throughout the description for this section.

Reply:
@LipingWang when you look through the Engineering Reference, K is much more likely to be used than C in the denominator of units. It would be good to clean this up.